googleMapsApiModule.py

```diff
@@ -20,7 +20,15 @@ def direction(origin,destination):
     logging.debug(result_url)
     webbrowser.open_new(result_url)
 
+def timezone(search_location,timestamp):
+    address = search_location.lower()
+    address = address.replace(" ","+")
+    result_url = f'https://maps.googleapis.com/maps/api/timezone/json?location={address}&timestamp={timestamp}&key={config.key}
```

Comment:
@hritikpapa Seems like this string is missing a quote `'` at the end. 
Also, can you share a test case which worked for you?

Reply:
I need API key to test it

Reply:
@hritikpapa the changes cannot be merged unless they are tested. Please test the code and share the screenshot of the tested code, and the corresponding test case.

Reply:
@vishakha-lall I will test it and share the screenshot with you :)

Reply:
@hritikpapa, please fix the missing quote at the end.

Reply:
I can fix it but I need to test this.